### ISSUE TITLE
denops.vim does not work in Deno1.36.2

### ISSUE THREAD

**Issue by kuuote** [vim-denops/denops.vim#272]
アップグレードすると死ぬので取り急ぎ
後で調査します
![Screenshot_20230822-085328](https://github.com/vim-denops/denops.vim/assets/36663503/7305f2d0-e434-4ecf-8b76-b9982e2ab9a2)

```
[denops] error
[denops] : 
[denops] Uncaught (in promise) Error: Failed to notify nvim_set_client_info("denops", {}, "remote", {"invoke":{"async":false,"nargs":2}}, {"website":"https://github.com/vim-denops/denops.vim","license":"MIT","logo":"https://github.com/vim-denops/denops-logos/blob/main/20210403-main/denops.png?raw=true"}): Error: Session is not running
[denops]       throw new Error(`Failed to notify ${method}(${paramsStr}): ${err}`);
[denops]             ^
[denops]     at Client.notify (https://deno.land/x/messagepack_rpc@v2.0.3/client.ts:131:13)
[denops]     at file:///tmp/dein/cache/hoge/.cache/init.vim/.dein/denops/@denops-private/host/nvim.ts:33:20
[denops]     at eventLoopTick (ext:core/01_core.js:183:11)
[denops] Server stopped (1). Restarting...
```

**Comment by kuuote** [comment]
VimでもNeovimでも関係なく起きるようだが、Vimではエラーが出ない

**Comment by Shougo** [comment]
https://github.com/denoland/deno/releases/tag/v1.36.2

release note

**Comment by Shougo** [comment]
Please set `denops#debug` and `denops#trace`.

**Comment by Shougo** [comment]
```
[denops] Connecting to channel `127.0.0.1:43085` [1/4]
[denops] Channel connected (127.0.0.1:43085)
[denops] DenopsReady
[denops] 1 plugins are discovered
[denops] register plugin: ['kensaku', '/home/shougo/.cache/dein/repos/github.com/lambdalisue/kensaku.vim/denops/kensaku/main.ts', {'platform': 'linux', 'host':
 'nvim', 'mode': 'debug', 'version': '0.10.0'}, {'mode': 'skip'}, v:true]
[denops] error: Uncaught (in promise) Error: Failed to notify nvim_set_client_info("denops", {"major":5,"minor":0,"patch":0,"prerelease":["20-g9f1b8cf"],"build
":[]}, "remote", {"invoke":{"async":false,"nargs":2}}, {"website":"https://github.com/vim-denops/denops.vim","license":"MIT","logo":"https://github.com/vim-den
ops/denops-logos/blob/main/20210403-main/denops.png?raw=true"}): Error: Session is not running
[denops]       throw new Error(`Failed to notify ${method}(${paramsStr}): ${err}`);
[denops]             ^
[denops]     at Client.notify (https://deno.land/x/messagepack_rpc@v2.0.3/client.ts:131:13)
[denops]     at file:///home/shougo/work/denops.vim/denops/@denops-private/host/nvim.ts:33:20
[denops]     at eventLoopTick (ext:core/01_core.js:183:11)
[denops] Server stopped: 1
[denops] Server stopped (1). Restarting...
[denops] Channel closed (127.0.0.1:43085)
[denops] DenopsClosed
[denops] Spawn server [1/4]
[denops] Server started: ['deno', 'run', '-q', '--no-lock', '--unstable', '-A', '/home/shougo/work/denops.vim/denops/@denops-private/cli.ts', '--quiet', '--ide
ntity', '--port', '0']
[denops] Server listen: 127.0.0.1:35781
[denops] Connecting to channel `127.0.0.1:35781` [1/4]
[denops] Channel connected (127.0.0.1:35781)
[denops] DenopsReady
[denops] 1 plugins are discovered
[denops] register plugin: ['kensaku', '/home/shougo/.cache/dein/repos/github.com/lambdalisue/kensaku.vim/denops/kensaku/main.ts', {'platform': 'linux', 'host':
 'nvim', 'mode': 'debug', 'version': '0.10.0'}, {'mode': 'skip'}, v:true]
[denops] error: Uncaught (in promise) Error: Failed to notify nvim_set_client_info("denops", {"major":5,"minor":0,"patch":0,"prerelease":["20-g9f1b8cf"],"build
":[]}, "remote", {"invoke":{"async":false,"nargs":2}}, {"website":"https://github.com/vim-denops/denops.vim","license":"MIT","logo":"https://github.com/vim-den
ops/denops-logos/blob/main/20210403-main/denops.png?raw=true"}): Error: Session is not running
[denops]       throw new Error(`Failed to notify ${method}(${paramsStr}): ${err}`);
[denops]             ^
[denops]     at Client.notify (https://deno.land/x/messagepack_rpc@v2.0.3/client.ts:131:13)
[denops]     at file:///home/shougo/work/denops.vim/denops/@denops-private/host/nvim.ts:33:20
[denops]     at eventLoopTick (ext:core/01_core.js:183:11)
[denops] Server stopped: 1
[denops] Server stopped 3 times within 10000 millisec. Denops is disabled to avoid infinity restart loop.
[denops] Channel closed (127.0.0.1:35781)
[denops] DenopsClosed
[denops] Failed to wait for "skkeleton" to start. Denops server itself is not started.
```

**Comment by Shougo** [comment]
```
[denops] Spawn server [1/4]
[denops] Server started: ['deno', 'run', '-q', '--no-lock', '--unstable', '-A', '/home/shougo/work/denops.vim/denops/@denops-private/cli.ts', '--quiet', '--ide
ntity', '--port', '0']
[denops] Server listen: 127.0.0.1:41521
[denops] Connecting to channel `127.0.0.1:41521` [1/4]
[denops] Channel connected (127.0.0.1:41521)
[denops] DenopsReady
[denops] 1 plugins are discovered
[denops] register plugin: ['kensaku', '/home/shougo/.cache/dein/repos/github.com/lambdalisue/kensaku.vim/denops/kensaku/main.ts', {'platform': 'linux', 'host':
 'vim', 'mode': 'debug', 'version': '9.0.1778'}, {'mode': 'skip'}, v:true]
[denops] Channel closed (127.0.0.1:41521)
[denops] DenopsClosed
```

It is Vim version.  Deno seems crashed.

**Comment by kuuote** [comment]
It causes by https://github.com/denoland/deno/commit/5834d282d4de5d0b5cacb9bf068f3896bef0a48a
